frontend/src/components/datasets/DatasetDetailContent.tsx

```diff
@@ -157,7 +193,7 @@ const DatasetDetailContent = () => {
     const extraButtons = <>
         <Button
             style={{ margin: 6 }}
-            onClick={(ev) => {
+            onClick={() => {
```

Comment:
The behaviour here is a bit strange for the user. Having to select Release all first to get access to Block all is not intuitive. (i know it is not new but it could be improved without doing too much.

Reply:
I've fixed that, so that both Release All and Block All are available.